common/build.gradle

```diff
@@ -73,13 +74,22 @@ android {
             outputFileName = "${archivesBaseName}-${version}.aar"
         }
     }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
 }
 
 dependencies {
     implementation "com.nimbusds:nimbus-jose-jwt:$rootProject.ext.nimbusVersion"
     implementation "androidx.appcompat:appcompat:$rootProject.ext.appCompatVersion"
     implementation "com.google.code.gson:gson:$rootProject.ext.gsonVersion"
     implementation "androidx.browser:browser:$rootProject.ext.browserVersion"
+    implementation "org.robolectric:annotations:$rootProject.ext.robolectricVersion"
+
+    implementation project(':keyvault')
```

Comment:
Agree with Shane we can't package these with `common` , can we only include it as ` testImplementation` as its only used for tests

Reply:
Addressed.

Moved all test classes to a separate module that is solely for test purposes. And then we include that module as testImplementation